### ISSUE TITLE
Lost focus event of GTextfield/GTextArea

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roloww/gui4processing#14]
```
What steps will reproduce the problem?
1.
2.
3.

What is the expected output? What do you see instead?
An event is fired when a GTextfield/GTextArea lost mouse focus.

What version of the product are you using? On what operating system?
g4p 3.1.1
processing 2.0b8

Please provide any additional information below.

```

Original issue reported on code.google.com by `phonghai...@gmail.com` on 19 Mar 2013 at 5:35

**Comment by GoogleCodeExporter** [comment]
```
Focus can be lost via the keyboard as well as the mouse.

If it is possible this will be introduced in the next version.
```

Original comment by `p.k.la...@gmail.com` on 19 Mar 2013 at 7:07
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
When a GTextField or GTextArea gains or looses focus then a GEevent.GETS_FOCUS 
or a GEvent.LOST_FOCUS event is fired.
This is available in the next release (3.2.4)
```

Original comment by `p.k.la...@gmail.com` on 12 Apr 2013 at 11:59
- Changed state: **Fixed**
